### ISSUE TITLE
Add quick start (introduction and usage) to README

### ISSUE THREAD

**Issue by jaredmorgs** [asciidoctor/asciidoctor-bespoke#1]
I seem to remember you tweeting _a lot_ about this as you were presenting at the various conferences. As someone who is totally new to this presentation framework, I wonder if a bit of quick-start README.adoc prose could be in order?

**Comment by mojavelinux** [comment]
Naturally. But it has to do something first :) I'm working on that part.

**Comment by mojavelinux** [comment]
I finally got a start on this. See https://github.com/asciidoctor/asciidoctor-bespoke#asciidoctor-bespoke

I discovered that we have a one major hurdle. This converter must be integrated into an existing Bespoke.js project. That means we need to find a way to get a beginner started with Bespoke.js using Yeoman. Right now, we have to modify the presentation that the generator-bespoke creates, which is less than ideal. What we need is a generated that creates a presentation with the converter preintegrated.

In other words, we need to create a new Yeoman generator :) It's not a showstopper for the alpha release, but definitely something that will be important to gain adoption.

**Comment by mojavelinux** [comment]
I think the README is now squared away (though the reference to the released gem won't work of course until the alpha is released). I also created a starter project to reduce the effort of bootstrapping a presentation.
